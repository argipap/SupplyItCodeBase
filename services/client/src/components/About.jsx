import React from 'react';

const About = () => (
    <div>
        <h1 className="title is-1">About</h1>
        <hr/><br/>
        <p>SupplyIt! No more orderings via telephone communications!</p>
    </div>
);

export default About;