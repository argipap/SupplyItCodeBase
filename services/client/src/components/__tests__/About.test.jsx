import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import About from '../About';

test('About renders properly', () => {
    const wrapper = shallow(<About/>);
    const element = wrapper.find('p');
    expect(element.length).toBe(1);
    expect(element.text()).toBe('SupplyIt! No more orderings via telephone communications!');
});

test('About renders a snapshot properly', () => {
    const tree = renderer.create(<About/>).toJSON();
    expect(tree).toMatchSnapshot();
});