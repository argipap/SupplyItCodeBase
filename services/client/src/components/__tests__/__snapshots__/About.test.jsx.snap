// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders a snapshot properly 1`] = `
<div>
  <h1
    className="title is-1"
  >
    About
  </h1>
  <hr />
  <br />
  <p>
    SupplyIt! No more orderings via telephone communications!
  </p>
</div>
`;
